frontend/src/components/SearchPage/CourseItem.js

```diff
@@ -203,17 +207,19 @@ const CourseItem = ({ name, code, professors, metrics, UID }) => {
             {code}
           </CourseCode>
           <CourseMetrics>
-            <CourseMetric>
-              {round(metrics.overall, 1)}<br/>
+            <CourseMetricReal>
+              {authed ? round(metrics.overall, 1) : 3.2}
```

Comment:
Hmm... I'm a little sketched out by hardcoding random values. From what I understand, the user won't be able to read these anyways through the CSS blur, but I think you should add a comment that describes this situation here so that in the future a contributor isn't thrown off by these numbers.

Reply:
Yeah maybe we use a call to random, and document it ofc

Reply:
I think just leaving value in is fine as long as there's a little comment